view/src/components/surplus.js

```diff
@@ -350,13 +408,23 @@ class Surplus extends Component {
     if (this.state.uiLoading === true) {
       return (
         <main className={classes.content}>
-          {this.state.uiLoading && <CardSkeletons classes={classes} />}
+          {this.state.uiLoading && (
+            <CardSkeletons classes={classes} noPadding={!this.props.main} />
+          )}
         </main>
       );
     } else {
       return (
         <main className={classes.content}>
-          <div className={classes.toolbar} />
+          {/* Only show if it is not the main page */}
+          {!this.props.main && (
```

Comment:
I think rather than calling this "main", it would be better to invert its meaning and come up with a meaningful name for the mode where this is acting as part of a flow.  I feel like "main" should be the default in the absence of other information.  It's kind of like if you walk into a restaurant that has special instructions for seating anyone who is not accompanied by an elephant.  You understand, but you have a lot of questions.